packages/builders/src/interactions/slashCommands/SlashCommandBuilder.ts

```diff
@@ -188,8 +188,7 @@ export class SlashCommandBuilder {
 export interface SlashCommandBuilder extends SharedNameAndDescription, SharedSlashCommandOptions {}
 
 export interface SlashCommandSubcommandsOnlyBuilder
-	extends SharedNameAndDescription,
-		Pick<SlashCommandBuilder, 'addSubcommand' | 'addSubcommandGroup' | 'toJSON'> {}
+	extends Omit<SlashCommandBuilder, Exclude<keyof SharedSlashCommandOptions, 'options'>> {}
```

Comment:
Does this only remove the option functions or does it keep them 👀

Reply:
It omits all the methods of the [`SharedSlashCommandOptions`](https://github.com/discordjs/discord.js/blob/d08a57cadd9d69a734077cc1902d931ab10336db/packages/builders/src/interactions/slashCommands/mixins/SharedSlashCommandOptions.ts#L17-L161) mixin from the [`SlashCommandBuilder`](https://github.com/discordjs/discord.js/blob/d08a57cadd9d69a734077cc1902d931ab10336db/packages/builders/src/interactions/slashCommands/SlashCommandBuilder.ts#L21-L22) class